main.py

```
@@ -4,7 +4,7 @@
 from langchain_core.messages import SystemMessage, HumanMessage, AIMessage
 from langchain_core.prompts import ChatPromptTemplate, MessagesPlaceholder
 import gradio as gr
-import utils
+from utils import *
```

Ruff (0.12.2)

`from utils import *` used; unable to detect undefined names

(F403)